ansible/ssh/known_hosts

```diff
@@ -2,4 +2,4 @@ cafe.privacylx.org,37.218.245.27 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHA
 mail.privacylx.org,37.218.245.28 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBEO3WIg89nI504rk8MGB6woBxHL3JVJDYheg3kgE9gtMnWocc6Ae85XV67XSHlAmaJO9TT144i0wb89v+MV4MuE=
 privacylx.org,37.218.245.53 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMf1ZuIh/5qXRAFUoMlK+SE0YhMKNKJsKZRrpkq7o2YsmnzSrxGZE4KbImjnoutCLxfqlMWfMcW6zGEwDFP1lvY=
 johndoe.privacylx.org,37.218.242.44 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBF6toPYWJuqj9Y+XZ6JnGtvMqzMu1q+UxtvkKXpGEb/m+wT2mZDRI6H0dA/tPQV1ZZ4//1Pl1+cieHSXNxvAHww=
-testing.privacylx.org,37.218.241.145 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBDok2hLGVfO4hKdHhXa1s4T1jh9p1mrV80yqEaGIzIfWOEMEFYn4badT8v689pwlgRW06Gp/0ThViMHihbmBssY=
+testing.privacylx.org,18.234.80.73 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBGj16FJNbwX+wkI7rLWsMXIDY0FgWrNM3A+MQ5HND4qzDrozg/yKHHMcsyJxjCMeskrwWqpNGbpP7jzRO+OeEv8=
```

Comment:
Wrong IP and SSH fingerprint?

Reply:
as you may recall we no longer have a testing server. So I had to boot my own and patch the `know_hosts` because we have `strict_host_verification`. I accidentally uploaded this. I'll remove the file

Reply:
There is a testing server since some time ago.
Please see https://github.com/PrivacyLx/devops/commit/157bfdfbcfccc7a8eb0190383348200d53e59c45 .

Reply:
Ah. I see. I forgot about that